pom.xml

```diff
@@ -91,88 +91,75 @@
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
 		</dependency>
-		
-		<dependency>
-			<groupId>io.springfox</groupId>
-			<artifactId>springfox-swagger2</artifactId>
-			<version>2.9.2</version>
-			<exclusions>
-				<exclusion>
-					<groupId>com.google.guava</groupId>
-					<artifactId>guava</artifactId>
-				</exclusion>
-			</exclusions>	
-		</dependency>
-		<dependency>
-		    <groupId>com.google.guava</groupId>
-		    <artifactId>guava</artifactId>
-		    <version>28.0-jre</version>
-		</dependency>
 		<dependency>
 			<groupId>org.springframework.data</groupId>
 			<artifactId>spring-data-commons</artifactId>
 			<version>${spring.data}</version>
 			<scope>provided</scope>
 		</dependency>
 
-<!-- START BOOMERANG LIBS -->		
-<!-- 	<dependency>
+		<!-- START BOOMERANG LIBS -->
+		<!-- 	<dependency>
 			<groupId>net.boomerangplatform</groupId>
 			<artifactId>lib-security</artifactId>
 			<version>1.0.3</version>
-		</dependency>-->		
+		</dependency>-->
 		<dependency>
 			<groupId>net.boomerangplatform</groupId>
 			<artifactId>lib-rest</artifactId>
 			<version>1.0.1</version>
```

Comment:
This library seems to be renamed 4 years ago and the last tag is 0.0.7, so I'm not sure where that 1.0.1 version is taken from. Might be a public repository?